jwst/regtest/test_nircam_coron3.py

```diff
@@ -12,7 +13,10 @@ def run_pipeline(jail, rtdata_module):
 
     # Run the calwebb_coron3 pipeline on the association
     args = ["calwebb_coron3", rtdata.input]
-    Step.from_cmdline(args)
+    with warnings.catch_warnings():
+        # warning is explicitly raised by the pipeline
+        warnings.filterwarnings("ignore", category=RuntimeWarning, message="var_rnoise array not available")
```

Comment:
So does this mean that users will no longer see this warning in the processing log? This one could be important to see, so that they understand that the error computations are potentially compromised due to the fact that there's no `var_rnoise` data. On the other hand, this probably only happens when running on really old test data that didn't yet have that data array added to it. All "real" datasets coming through now should always have that data array. So maybe it's not a big deal after all.

Reply:
This filter is within a test, so it does not affect what a user would see, just someone running the test suite.  And I assumed that we were not worried about these warnings in the context of what this test attempts to test.  But I'm happy to remove it again if you like.